server/profile.js

```diff
@@ -42,6 +48,13 @@ module.exports.getProfile = async (event) => {
     }
   }
 
+  const selectRefs = "SELECT * FROM has_ref LEFT JOIN reference ON has_ref.ref_id = reference.ref_id WHERE has_ref.profile_id = $1";
+  const values2 = [profileId]
```

Comment:
why is the profileId in square brackets?

Reply:
I followed the format of the other SELECT statement, but I believe because it should be provided as an array (this is me inferring though)

Reply:
Not necessarily... JS is really smart at inferring the appropriate data type. I don't mind leaving it in an array jic ion the future the SQL stmt may change. But if its only value, you could have passed it directly into query(), FYI :)